unitTests/ui/ResultListTest.ts

```diff
@@ -784,11 +784,48 @@ export function ResultListTest() {
         describe('enableScrollToTop set to', () => {
           let infiniteScrollContainer: HTMLElement;
           let allResult: IQueryResults;
+          let scrollToSpy: jasmine.Spy;
+          let scrollTopSpy: jasmine.Spy;
+
           beforeEach(() => {
-            infiniteScrollContainer = { scrollTop: 500 } as HTMLElement;
+            document.createElement('div');
+            infiniteScrollContainer = document.createElement('div');
+            infiniteScrollContainer.style.height = '10000px';
+            infiniteScrollContainer.style.overflow = 'scroll';
+            document.body.appendChild(infiniteScrollContainer);
+            scrollToSpy = spyOn(infiniteScrollContainer, 'scrollTo');
+            let pValue = 500;
+            scrollTopSpy = jasmine.createSpy('scrollTop', v => (pValue = v));
+            Object.defineProperty(infiniteScrollContainer, 'scrollTop', {
+              get: () => pValue,
+              set: scrollTopSpy
+            });
             allResult = FakeResults.createFakeResults(50);
           });
 
+          afterEach(() => {
+            document.body.removeChild(infiniteScrollContainer);
+          });
+
+          it('should scroll to top on query success even in IE11', done => {
+            infiniteScrollContainer.scrollTo = undefined;
+            const option: IResultListOptions = {
+              enableInfiniteScroll: true,
+              infiniteScrollContainer: infiniteScrollContainer,
+              enableScrollToTop: true
+            };
+            test = Mock.basicComponentSetup<ResultList>(ResultList, option);
+            test.cmp.currentlyDisplayedResults.push(allResult.results[1]);
+            (test.cmp.queryController.fetchMore as jasmine.Spy).and.returnValue(Promise.resolve(allResult));
```

Comment:
I'm reading the tests in this file, and it seems this setup gets repeated quite a bite.

Would it be possible to modify the tests to group them together in one describe block with the same `beforeEach()` setup instead ?

Reply:
Yep will do